src/Core/Services/StateService.cs

```diff
@@ -1538,10 +1538,7 @@ private async Task CheckStateAsync()
             if (!_migrationChecked)
             {
                 var migrationService = ServiceContainer.Resolve<IStateMigrationService>("stateMigrationService");
```

Comment:
Since we're changing stuff, could you remove the string key of the service Resolve here?